app/frontend/src/views/admin/Submission.vue

```diff
@@ -77,7 +81,23 @@ export default {
     return {
       error: false,
       ipcPlanData: {},
-      showFv: false
+      showFv: false,
+      // quick fix for getting display name
+      operationTypes: [
+        { type:'BRUSHING', display: 'Brushing', enabled: true },
+        { type:'CONE_PICKING', display: 'Cone Picking', enabled: true },
+        { type:'ENGINEERING', display: 'Engineering', enabled: true },
+        { type:'LAYOUT', display: 'Layout/Timber Cruising', enabled: true },
+        { type:'LOGGING', display: 'Logging', enabled: true },
+        { type:'PRUNING', display: 'Pruning', enabled: true },
+        { type:'RESEARCH', display: 'Research', enabled: true },
+        { type:'SILVICULTURE', display: 'Silviculture', enabled: true },
+        { type:'SPACING', display: 'Spacing', enabled: true },
+        { type:'SURVEYING', display: 'Surveying', enabled: true },
+        { type:'VEGETATION_MANAGEMENT', display: 'Vegetation Management', enabled: true },
+        { type:'WILDFIRE_CAMPS', display: 'Wildfire Camps', enabled: true },
+        { type:'OTHER', display: 'Other', enabled: true }
```

Comment:
Duplicate on the public/staff, so should be a constant to make sure no risk of bugs if this list is modified. I suppose this is a little simpler in the Comfort version since the whole defined object is returned from the API, whearas in this one the API only knows about the type right?

Reply:
yes the api only returns the 'type' from the database. i'll put this in a constants file.